container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3057,7 +3049,7 @@ configuration:
     # TODO: this needs to be fixed upstream - it should be an array
     properties.opi.nats_ip: "nats-0.nats-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))"
     properties.opi.nats_password: "((NATS_PASSWORD))"
-    properties.opi.registry_address: "registry.((DOMAIN))"
+    properties.opi.registry_address: "127.0.0.1:((OPI_REGISTRY_NODEPORT))"
```

Comment:
Ah because we don't use bits here anymore?

Reply:
Yes